README.md

```diff
@@ -1,384 +1,212 @@
+        let ul = document.createElement('ul')
+            for (let obj of results) {
+                let li = document.createElement('li')
+                li.innerText = obj.name + " " + obj.population 
+                ul.appendChild(li)
+            }
+            document.body.appendChild(ul)
+        </script>
+
+    </body>
+</html>
 
-# Question Five
-
-Make a website with a text input and a button.  Clicking on the button should create a new list item with its text the same as the text in the input.
-
-# Question Six
-
-Make a website with a text input, a number input and a button.  Clicking on the button should add that many copies of the text in the text input to a list.
+```
 
-# Question Seven
+Question Five
+Make a website with a text input and a button. Clicking on the button should create a new list item with its text the same as the text in the input.
 
-Make a website with a number input, a button, and 3 paragraphs.  Clicking on the button should add the number to a list.  The first paragraph should show the average of all the numbers, the second paragraph should show the maximum value, and the third paragraph should show the minimum value.
+```
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+
+        <label for="list_entry">Enter an item for your list: </label>
+        <input type="text" name="list_item" id="list_entry">
+        <button
+            id="add_entry"
+            name="list_item"
+            onclick="addItem()"
+        >
+            Add an Item to the List
+        </button>
+        <ul id="entries"></ul>
+        <script>
+        const addItem = () => {
+            let item = document.createElement('li')
+            item.innerText = document.querySelector('#list_entry').value
+            document.querySelector('#entries').appendChild(item)
+        }
+        
+        </script>
+    </body>
+</html>
 
-# Question Eight
+```
+Question Six
+Make a website with a text input, a number input and a button. Clicking on the button should add that many copies of the text in the text input to a list.
 
-Make a website with a button.  Clicking on the button should remove the button.
+```
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+      
+        <label for="input_text">Enter your text here</label>
+        <br>
+        <input type="text" name="text_entry" id="input_text">
+        <br>
+        <label for="amount">Enter amount of times</label>
+        <br>
+        <input type="number" name="text_entry" id="amount">
+        <br>
+        <button
+            id="add_entry"
+            name="text_entry"
+            onclick="dispXTimes()"
+        >
+            Add
+        </button>
+        <p id="entries"></p>
+        <script>
+            const dispXTimes = () => {
+                for (let i = 0; i < document.querySelector('#amount').value; i++) {
+                    document.querySelector('#entries').innerText += `${document.querySelector('#input_text').value}\n`
```

Comment:
It would be better if u created a variable to be the entries and then added the innerText to that variable.

What you have here is valid but it is just more reabable